src/__tests__/Pages.test.tsx

```diff
@@ -146,6 +146,7 @@ const signupGroupValues: SignupGroupFormFields = {
       id: TEST_SIGNUP_ID,
       inWaitingList: false,
       lastName: 'Last name',
+      phoneNumber: '0441234567',
```

Comment:
Perhaps the additions of `phoneNumber` should be in a commit of their own

Reply:
Good point. Moved `phoneNumber` additions to a separate commit